.golangci.yml

```diff
@@ -64,3 +64,40 @@ run:
   # Mainly related to generics support since go1.18.
   # Default: use Go version from the go.mod file, fallback on the env var `GOVERSION`, fallback on 1.18
   go: '1.16'
+
+linters:
+  disable:
+    - varnamelen
```

Comment:
@titpetric can we add some comments on why we are disabling those?

Reply:
@tbuchaillot all in JIRA ticket + the RFC on confluence :)